### ISSUE TITLE
[BUG] Bulk actions on the Plugins page are hidden

### ISSUE THREAD

**Issue by pryley** [robertdevore/benchpress#3]
This is happening because you are loading the benchpress assets on every admin screen.

Possible solutions are:

### 1. Only load the benchpress assets on benchpress admin pages (unless they are needed globally).

For example:

```php
function benchpress_enqueue_assets(): void
{
    if (!str_contains(get_current_screen()->base, 'page_benchpress')) {
        return;
    }
    // enqueue assets here...
}
add_action('admin_enqueue_scripts', 'benchpress_enqueue_assets');
```

### 2. Override the protected `display_tablenav` method in your extended `WP_List_Table` classes.

For example:

```php
protected function display_tablenav($which)
{
    if ('top' === $which) {
        return;
    }
    parent::display_tablenav($which);
}
```

### 3. Prefix your CSS selectors with the page specific body class.

For example:

```css
[class*="page_benchpress"] .tablenav.top {
    display: none;
}
```

**Comment by robertdevore** [comment]
Thank you! I thought I had that check in place already but apparently not 🤦‍♂️

Pushing out an update this evening 🤘

**Comment by robertdevore** [comment]
@pryley I have pushed out a fix for this in v1.0.2

I'm closing this out, but feel free to re-open this issue or create a new one if anything else comes up 🤘
